app/views/evaluation_forms/_form.html.erb

```diff
@@ -22,7 +22,7 @@
           <% current_user.challenge_manager_challenges.each do |challenge| %>
             <% if challenge.is_multi_phase %>
               <% challenge.phases.each.with_index(1) do |phase, index| %>
-                <option value="<%= "#{challenge.id}.#{index}.#{phase.end_date.strftime("%m/%d/%Y")}" %>"><%= challenge.title %> - Phase <%= index %></option>              
+                <option value="<%= "#{challenge.id}.#{phase.id}.#{phase.end_date.strftime("%m/%d/%Y")}" %>"><%= challenge.title %> - Phase <%= index %></option>              
```

Comment:
I think the title could be extracted to a reusable function similar to `challenge_with_phase` for easier reuse